packages/enmeshed_types/lib/src/contents/request_items/authentication_request_item.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:json_annotation/json_annotation.dart';
+
+import 'request_item_derivation.dart';
+
+part 'authentication_request_item.g.dart';
+
+@JsonSerializable(includeIfNull: false)
+class AuthenticationRequestItem extends RequestItemDerivation {
+  const AuthenticationRequestItem({super.title, super.description, super.metadata, required super.mustBeAccepted, super.requireManualDecision});
+
+  factory AuthenticationRequestItem.fromJson(Map json) => _$AuthenticationRequestItemFromJson(Map<String, dynamic>.from(json));
+
+  @override
+  Map<String, dynamic> toJson() {
+    final json = super.toJson();
+    json.addAll(_$AuthenticationRequestItemToJson(this));
+    json['@type'] = 'AuthenticationRequestItem';
+    return json;
+  }
```

Comment:
this shoule be done in the darty way:

```suggestion
  Map<String, dynamic> toJson() => { ...super.toJson(), ..._$AuthenticationRequestItemToJson(this), '@type': 'AuthenticationRequestItem' }
```

Reply:
I also think that super.toJSON is called under the hood in the generated method.

Reply:
If I remove `...super.toJson()`, it says `This method overrides a method annotated as '@mustCallSuper' in 'RequestItem', but doesn't invoke the overridden method`.

Reply:
hmm, I see.